designer/server/src/test/scala/pl/touk/nussknacker/ui/api/DictsResourcesTest.scala

```diff
@@ -0,0 +1,176 @@
+package pl.touk.nussknacker.ui.api
+
+import com.typesafe.config.Config
+import io.circe.Json
+import io.circe.syntax.EncoderOps
+import org.scalatest.funsuite.AnyFunSuiteLike
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.engine.api.typed.typing.Typed
+import pl.touk.nussknacker.test.WithTestHttpClient
+import pl.touk.nussknacker.test.base.it.NuItTest
+import pl.touk.nussknacker.test.config.WithSimplifiedDesignerConfig.TestProcessingType.Streaming
+import pl.touk.nussknacker.test.config.{ConfigWithScalaVersion, WithDesignerConfig}
+import pl.touk.nussknacker.ui.api.DictResourcesEndpoints.Dtos.{DictListRequestDto, TypingResultInJson}
+import sttp.client3.{UriContext, quickRequest}
+import sttp.model.{MediaType, StatusCode}
+
+class DictsResourcesTest
+    extends AnyFunSuiteLike
+    with NuItTest
+    with WithDesignerConfig
+    with WithTestHttpClient
+    with Matchers {
+
+  override def designerConfig: Config = ConfigWithScalaVersion.TestsConfigWithEmbeddedEngine
+
+  test("return list of available dictionaries for DictParameterEditor - String") {
+    val response = httpClient.send(
+      quickRequest
+        .post(
+          uri"$nuDesignerHttpAddress/api/processDefinitionData/${Streaming.stringify}/dicts"
+        )
+        .contentType(MediaType.ApplicationJson)
+        .body(
+          DictListRequestDto(TypingResultInJson(Typed[String].asJson)).asJson.spaces2
```

Comment:
Please use literal text. Currently I can't see what exactly we want from the client to pass here. Take a look how we do this in other tests.